components/dropdown/dropdown-button.js

```diff
@@ -0,0 +1,64 @@
+import '../button/button.js';
+import '../icons/icon.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { DropdownOpenerMixin } from './dropdown-opener-mixin.js';
+import { dropdownOpenerStyles } from './dropdown-opener-styles.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+class DropdownButton extends DropdownOpenerMixin(RtlMixin(LitElement)) {
+
+	static get properties() {
+		return {
+			primary: {
+				type: Boolean,
+				reflect: true
+			},
+			text: {
+				type: String
+			}
+		};
+	}
+
+	static get styles() {
+		return [dropdownOpenerStyles, css`
+			d2l-icon {
+				height: 0.8rem;
+				width: 0.8rem;
+			}
+			.d2l-dropdown-button-text {
+				margin-right: 0.6rem;

```

Comment:
Hmm, I wonder why we didn't simplify this and just put the margin on the icon, removing the need for this span...